guides/doc-Administering_Red_Hat_Satellite/topics/Disabling_Weak_Encryption.adoc

```diff
@@ -24,28 +24,16 @@ To disable weak encryption for {Project}, complete the following steps:
 +
 [options="nowrap"]
 ----
-# Foreman Proxy
-foreman_proxy::tls_disabled_versions: [ '1.1' ]
-
-# Dynflow
-foreman_proxy::plugin::dynflow::tls_disabled_versions: [ '1.1' ]
-
 # Apache
 apache::mod::ssl::ssl_protocol: [ 'ALL' , '-SSLv3' , '-TLSv1' , '-TLSv1.1' , '+TLSv1.2' ]
-
-# Tomcat / Candlepin
-candlepin::tls_versions: [ '1.2' ]
-
-# QPID Dispatch
-foreman_proxy_content::qpid_router_ssl_protocols: [ 'TLSv1.2' ]
-foreman_proxy_content::qpid_router_ssl_ciphers: 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'
 ----
 +
-. Rerun the `{foreman-installer}` tool with no arguments:
+. Enter the `{foreman-installer}` command with the following arguments:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer}
+# {foreman-installer} --foreman-proxy-tls-disabled-versions=1.1 \
```

Comment:
TLS 1.1 is gone since Foreman Proxy 2.1 (https://github.com/theforeman/smart-proxy/commit/ebdde78c329d6bf67b198446dae64d8952d0ac4a). Please drop this.

Reply:
The option `foreman-proxy-tls-disabled-versions=1.1` disables TLS version 1.1. From the help output:
`--foreman-proxy-tls-disabled-versions  List of TLS versions that will be disabled from the default`

Reply:
But it's redundant. If you don't specify it, the Smart Proxy won't use it either. The only use it has now is to disable TLS 1.2 and only use TLS 1.3, but that's not really relevant for most deployment.

Reply:
Got it, thank you for the explanation. What with Satellites that exist longer, say for those who upgrade from 6.4. Will TLS 1.1 be enabled on them? Does installer disable TLS 1.1 when upgrading?

Reply:
It will still configure it but the smart proxy application **always** disables some protocols:
https://github.com/theforeman/smart-proxy/blob/7c818d0cf1606a7d16c8334ab4fd256370eac365/lib/launcher.rb#L73-L79
Currently that's SSLv2, SSLv3, TLS 1.0 and (since Foreman 2.1 / Satellite 6.8) TLS 1.1. It then parses the disabled option:
https://github.com/theforeman/smart-proxy/blob/7c818d0cf1606a7d16c8334ab4fd256370eac365/lib/launcher.rb#L81-L90
So effectively what happens is:
```ruby
options |= OpenSSL::SSL::OP_NO_TLSv1_1 # hardcoded
options |= OpenSSL::SSL::OP_NO_TLSv1_1 # via options
```

Since it doesn't do anything, I didn't bother cleaning up options in a migration. The user will still only use strong encryption.

Reply:
Thank you for the clarification, I have removed this option.